### PR TITLE
fix: remove grey and white from tagColors

### PR DESCRIPTION
**Pull request by elizabethhalper**
Tags with grey and white colors associated with them are not recognized by canvas tools and therefore are rendered in the editorPage as red. Eliminating these colors to fix the bug.

### CODE REVIEW

File: src/react/components/common/tagColors.json [hunk 1]
```diff
@@ -1,6 +1,4 @@
 [
-    "#FFFFFF",
```
Comment (tbarlow12) [inline src/react/components/common/tagColors.json]: I think we actually might be able to get rid of this file entirely now. I had forgotten that it was still there

Comment (tbarlow12) [reply src/react/components/common/tagColors.json]: If not, we should update to use the object from `tagColors.ts`

Comment (elizabethhalper) [reply src/react/components/common/tagColors.json]: I'll try getting rid of it and see if I run into any issues

Comment (wbreza) [reply src/react/components/common/tagColors.json]: @JacopoMangiavacchi is using the colors somewhere in VoTT. I believe within the TF records

File: package.json [hunk 2]
```diff
@@ -43,7 +43,7 @@
         "shortid": "^2.2.14",
         "video-react": "^0.13.2",
         "vott-ct": "^2.1.19",
-        "vott-react": "^0.2.8"
+        "vott-react": "^0.2.9"
```
Comment (wbreza) [inline package.json]: Looks like package-lock is causing a failure... Not sure how you updated this.  Was it manual? or via `npm install`?  You will have to run a real `npm install` for the package lock to get updated correctly.

File: src/react/components/common/tagColors.json (hunk 1 above)
Comment (JacopoMangiavacchi) [reply src/react/components/common/tagColors.json]: Yes I'm using this when importing TFRecords.  In that scenario we are importing potentially new tag definitions that have to be added to IProject.ITags list.  Basically I need to associate a new color from that list to the new ITag object.

**Pull request merged**